package-overrides/npm/ng-clip@0.2.6.json

```diff
@@ -0,0 +1,15 @@
+{
+  "registry": "jspm",
+  "directories": {
+    "lib": "src"
+  },
```

Comment:
We discourage the use of `directories.lib` as it can be breaking for existing users of the package.